cmd/main.go

```diff
@@ -18,11 +18,11 @@ func main() {
 
 	dbUser := os.Getenv("DB_USER")
 	dbPassword := os.Getenv("DB_PASSWORD")
-	dbName := os.Getenv("DB_NAME")
-
```

Comment:
I would rather continue using the env var instead of hard coding the db name.

Reply:
Thanks for the feedback! I will reintroduce the environment variable `DB_NAME`.